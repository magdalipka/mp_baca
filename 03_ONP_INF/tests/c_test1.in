4
ONP: abcd*-/bx^+
ONP: abc*-db/x^+
ONP: abc*-dbx/^+
ONP: ab~+~