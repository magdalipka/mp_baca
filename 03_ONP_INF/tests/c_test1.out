INF: a / ( b - c * d ) + b ^ x
INF: a - b * c + ( d / b ) ^ x
INF: a - b * c + d ^ ( b / x )
INF: ~ ( a + ~ b )